dual_net.py

```diff
@@ -305,30 +293,31 @@ def export_model(working_dir, model_path):
         tf.gfile.Copy(filename, destination_path)
 
 
-def train(working_dir, tf_records, steps=None, **hparams):
-    estimator = get_estimator(working_dir, **hparams)
+def train(working_dir, tf_records, steps=None):
+    estimator = get_estimator(working_dir)
 
     def input_fn():
-        return preprocessing.get_input_tensors(TRAIN_BATCH_SIZE, tf_records,
-                                               filter_amount=1.0)
+        return preprocessing.get_input_tensors(
+            FLAGS.train_batch_size, tf_records, filter_amount=1.0,
+            shuffle_buffer_size=FLAGS.shuffle_buffer_size)
```

Comment:
This is an intentional change from 2,000,000 to 10,000. Given that you are now using the example buffer it should not affect training. But if you'd be more comfortable keeping it at 2M, I can do that.

Reply:
I'll change to 2M since this is merely intended to be a refactor, not change in functionality. Please feel free to drop this to a faster-filling number as you see fit; it should not affect the shuffling characteristics.

Reply:
ok, thanks!